ApollosProject/apollos-prototype
# Fixes email field rendering as [Object object] for sms users

## DESCRIPTION

### What does this PR do, or why is it needed?

When a user logs in w/ SMS, their account will most likely not have an email.
When they visit the profile screen, their email will show up as `[Object object]` because we don't protect against null values (which rock returns as `{}` in some instances)

This PR generalizes the null value check, and applies it to the Person's email field

### What design trade-offs/decisions were made?

### How do I test this PR?

- Login via SMS
  - (Currently impossible with the SMS UI Auth PR merged)
- Ensure your email field is blank in the settings, rather than `[Object object]`.

### What automated tests did you write?

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
